test/core/formattersTests.ts

```diff
@@ -168,6 +168,27 @@ describe("Formatters", () => {
     });
   });
 
+  describe("shortScale", () => {
+    it("shortens long numbers", () => {
+      var fmtr = Plottable.Formatters.shortScale();
+      assert.strictEqual(fmtr(1), "1.000", "small numbers format simply");
+      assert.strictEqual(fmtr(-1), "-1.000", "negative small numbers format simply");
+      assert.strictEqual(fmtr(2e3), "2.000K", "thousands numbers format using short scale magnitudes");
+      assert.strictEqual(fmtr(2e6), "2.000M", "millions numbers format using short scale magnitudes");
+      assert.strictEqual(fmtr(2e9), "2.000B", "billions numbers format using short scale magnitudes");
+      assert.strictEqual(fmtr(2e12), "2.000T", "trillions magnitude numbers format using short scale magnitudes");
+      assert.strictEqual(fmtr(2e15), "2.000Q", "quadrillions numbers format using short scale magnitudes");
```

Comment:
Our max is 10^(3*5+3+1) = 10^19. 

Can we try 
- 10^19
- 10^19 - 1
- 10^18 ?